website/src/components/wordpress.tsx

```diff
@@ -87,10 +107,74 @@ const parseOptions: HTMLReactParserOptions = {
   },
 }
 
-const pullWords = (
-  docName: string,
-  start: string,
-  end?: string
-): JSX.Element => {
-  return <div>full array</div>
+const PullWords = (props: { slug: string; start: number; end?: number }) => {
+  const slug = props.slug
+  const start = props.start
+
+  var end
+  end = props.end ? props.end : start + 1
```

Comment:
Since end doesn't get assigned again, I think making a single line const would work.

Reply:
this clause allows for ease of understandability of the component

Reply:
@KushP10 I don't know what that means. The comment was just to condense this declaration into a `const` since you don't mutate the variable. Does that seem less readable or something?

Reply:
> @KushP10 I don't know what that means. The comment was just to condense this declaration into a `const` since you don't mutate the variable. Does that seem less readable or something?

Sorry, I misinterpreted what Shashwat's comment meant.